scm/driver/bitbucket/webhook.go

```diff
@@ -46,24 +46,29 @@ func (s *webhookService) Parse(req *http.Request, fn scm.SecretFunc) (scm.Webhoo
 		hook, err = s.parsePullRequestHook(data)
 	case "pullrequest:updated":
 		hook, err = s.parsePullRequestHook(data)
-		if hook != nil {
-			hook.(*scm.PullRequestHook).Action = scm.ActionSync
-		}
+		// if hook != nil {
+		// 	hook.(*scm.PullRequestHook).Action = scm.ActionSync
+		// }
+		hook.(*scm.PullRequestHook).Action = scm.ActionSync
```

Comment:
`hook != nil`, this is always true, that means we always run `hook.(*scm.PullRequestHook).Action = scm.ActionSync` so I think we should just removing the `if-statement`.